src/containers/pages/IRS/JurisdictionsReport/tests/helpers.test.ts

```diff
@@ -0,0 +1,51 @@
+import * as helpers from '../helpers';
+import * as fixtures from './fixtures';
+
+describe('NamibiaColumns', () => {
```

Comment:
Instead of just `NamibiaColumns` could we have something like `JurisdictionsReport/helpers/NamibiaColumns/`?